src/gui/gui.zig

```diff
@@ -568,6 +568,9 @@ pub fn updateAndRenderGui() void {
 }
 
 pub fn toggleGameMenu() void {
+	if(hideGui) {
+		hideGui = false;
```

Comment:
I think it should only do this when the menu wasn't open already.
Also the if is kind of unnecessary.